skype/skype-ps/skype/Set-CsTeamsMessagingPolicy.md

```diff
@@ -204,6 +204,7 @@ Accept wildcard characters: False
 
 ### -AllowUserDeleteMessage
 Determines whether a user is allowed to delete their own messages. Set this to TRUE to allow. Set this to FALSE to prohibit.
+If this value is set to FALSE, the team owner will not be able to delete his own messages.
```

Comment:
So even if AllowOwnerDeleteMessage is true? Thanks for clarifying

Reply:
even if allowownerdeletemessage is on true the teams owner will not be able to delete the messages his sent in a team channel